packages/kbn-securitysolution-io-ts-types/src/iso_date_string/index.ts

```diff
@@ -13,6 +13,7 @@ import { Either } from 'fp-ts/lib/Either';
  * Types the IsoDateString as:
  *   - A string that is an ISOString
  */
+export type IsoDateString = t.TypeOf<typeof IsoDateString>;
```

Comment:
I think both `IsoDateString` and `IsoDateStringC` represent the same type, we can use either of them interchangeably, I wonder if is there any reason behind introducing this new `export type IsoDateString = t.TypeOf<typeof IsoDateString>;` ?

Reply:
@WafaaNasr I think ideally we should delete `IsoDateStringC` and use `IsoDateString` as the type instead. TypeScript aliases allow to have a type and a constant with the same name -- this is useful when they represent conceptually the same thing, in this case, a schema and its corresponding type.